.github/workflows/run-tests.yml

```diff
@@ -19,27 +18,96 @@ jobs:
 
     - name: Create .env file
       run: |
-        echo "REDIS_URL=redis://redis" >> .env
```

Comment:
@benjigifford You already have .env.test added to the repository, so you better use the same .env.test in tests for consitency.